File: app/src/main/java/com/github/nickpakhomov/hotel_viewer/util/image/ImageLoader.java
package com.github.nickpakhomov.hotel_viewer.util.image;

import android.widget.ImageView;

import com.squareup.picasso.Transformation;

/**
 * Created by Nikolay Pakhomov on 08/09/17.
 */

interface ImageLoader {
    void load(ImageView view, String url);
    
    void loadWithTransformation(ImageView view, String url, Transformation transformation);
}

File: app/src/main/java/com/github/nickpakhomov/hotel_viewer/util/image/ImageLoader.kt
package com.github.nickpakhomov.hotel_viewer.util.image

import android.widget.ImageView

import com.squareup.picasso.Transformation

/**
 * Created by Nikolay Pakhomov on 08/09/17.
 */

internal interface ImageLoader {
    fun load(view: ImageView, url: String)

    fun loadWithTransformation(view: ImageView, url: String, transformation: Transformation)
}

